docs/reference/tctl/workflow/show.md

```diff
@@ -0,0 +1,101 @@
+---
+id: show
+title: tctl workflow show
+description: How to show Workflow History using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../../../content/what-is-a-workflow-execution.md'
+import * as WhatIsAWorkflowId from '../../../content/what-is-a-workflow-id.md'
+import * as WhatIsARunId from '../../../content/what-is-a-run-id.md'
+
+The `tctl workflow show` command shows History for the specified <preview page={WhatIsAWorkflowExecution}>Workflow Execution</preview>.
+
+**Syntax** `tctl workflow show <command options> <arguments...>`
+
+### --workflow_id
```

Comment:
We could consider using backticks here:
```suggestion
### `--workflow_id`
```

Reply:
Because of the amount of inline code already on the page, I recommend not backticking* the H3s. The slightly larger, bold font for the H3s (plus the extra leading above and the leading double hyphens) visually demarcate each section.

*Totally not in the style guide. Totally oughta be.

Reply:
Because of other design changes that we made, I'm now in favor of backticking the H3s. (I'm also disavowing my use of "backticking.")